flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -91,18 +96,37 @@ class SentryFlutterOptions extends SentryOptions {
         : _cacheDirSize;
   }
 
+  @Deprecated('Use enableAppLifecycleBreadcrumbs instead')
+  bool get enableLifecycleBreadcrumbs => _enableAppLifecycleBreadcrumbs;
+
+  @Deprecated('Use enableAppLifecycleBreadcrumbs instead')
```

Comment:
I liked that, so it's just forwarding the options, but not breaking it.

Reply:
yep, would be bad to have a breaking change just after a major release.
It would be nice if it includes when it gets removed. So that users get nudged into using the superseding method.
`Use enableAppLifecycleBreadcrumbs instead. This option gets removed in Sentry 5.0.0` or something like that. What do you think?

Reply:
if they are using `pedantic` or something similar, they'd be warned by the linter (because of `Deprecated`), right? I guess this would be already good enough.
but not sure if I fully understand your suggestion, what do you mean by:

> It would be nice if it includes when it gets removed

Reply:
I think it would be nice to let the user know when this options get removed. Just like the example above:
`Use enableAppLifecycleBreadcrumbs instead. This option gets removed in Sentry 5.0.0`

Reply:
oh I see, so just a better message you mean, sounds good to me.
On Java, we have a specific annotation for this: https://github.com/JetBrains/java-annotations/blob/master/common/src/main/java/org/jetbrains/annotations/ApiStatus.java#L75-L92

if we don't have something similar on the `meta` pkg, a message is good enough :)

Reply:
Neither [dart:core](https://api.dart.dev/stable/2.10.4/dart-core/Deprecated-class.html) nor [meta](https://pub.dev/documentation/meta/latest/meta/meta-library.html) has something like that.

Is `5.0.0` a good target?

Reply:
yep, sounds good, thanks for that!